molecule/docker/tests/test_default.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+import os
+import yaml
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+    with open("../../defaults/main.yml", 'r') as stream:
+        return yaml.load(stream)
+
+
+@pytest.mark.parametrize("dirs", [
```

Comment:
This should match directories set in `molecule/docker/playbook.yml` with variables such as `prometheus_db_dir` and `prometheus_config_dir`